question1/README.md

```diff
@@ -2,5 +2,6 @@
 (K&R Exercise 5-2) Write <code>getfloat</code>, the floating-point analog of <code>getint</code>. What type does <code>getfloat</code> return as its function value? Verify that your function works with at least three examples.
 
 Compile Steps: 
+gcc q1.c
```

Comment:
output -1; you should print out the float to show that it's working.
output: 9
quality:9
18/20